### ISSUE TITLE
Better symlink handling

### ISSUE THREAD

**Issue by amyreese** [Instagram/Fixit#239]
If the files being linted involve symlinks somewhere between the root and the target, it's possible for the `relative_to` check in config to fail, resulting in an uncaught exception.

**Comment by michael-pplx** [comment]
From my testing, changing `.resolve()` to `.absolute()` resolved my issues. This is a problem for any repo using Bazel since the `bazel-bin` directory is symlinked.

**Comment by thatch** [comment]
@michael-pplx are you running fixit on something in bazel-bin?

**Comment by michael-pplx** [comment]
No, but the `bazel-bin` path gets resolved before the code that checks it against `.gitignore`.
